multihost_test/bz_automation/test_automation.py

```diff
@@ -127,3 +127,27 @@ def test_2091062(self, multihost, create_localusers):
         client.logout()
         assert "pam_motd: error scanning directory" not in \
                execute_cmd(multihost, "cat /var/log/secure").stdout_text
+
+    def test_sssd_6788(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: Add support for bcrypt password hashes for local users
+        :id: f25c7438-6db5-11ee-a613-845cf3eff344
+        :bugzilla: https://issues.redhat.com/browse/SSSD-6788
```

Comment:
Replace the reference to the [bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2218330) or [RHEL Jira](https://issues.redhat.com/browse/RHEL-5057) tickets